### DISCUSSION TITLE
Keyball44で左手のジャンパをショートさせるのに失敗しパッドが剥離してしまった

### DISCUSSION THREAD

**Discussion by Zazagann** [Yowkees/keyball#434] [Q&A]
お世話になっております。初めて自作キーボードを作成しているはんだ付け素人の者です。

Keyball44で左手のジャンパ「LEFT」をブリッジしてショートさせるのに失敗しパッドが剥離してしまいました。
以前の質問「[左手用デバイスのランドが剥がれたと思われます。対処法をご教授願います。](https://github.com/Yowkees/keyball/discussions/213)」にて解説されているKeyball61の基板の配置と異なるため、改めて対処法をお聞きしたいです。
また、それに関連してか、左のキーボードの一番左の列のキー（「Esc」「Tab」「Shift」）が反応しないです。
関連があるのか、それとも何か別の問題で対処法はあるのか、教えていただきたいです。

お忙しいところ恐れ入りますがご対応の程よろしくお願いします。
![IMG_3074](https://github.com/Yowkees/keyball/assets/63691865/54d5ba95-83b7-4546-9abd-245fcaa9e178)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

LEFT判定部分のパッド剥離ということで了解しました。
添付させていただいた写真の赤点どうしをエナメル線かゴム被覆線でつないでいただきますと、LEFT判定ジャンパをショートしたことになりますので、お手数ですがご対応をお願いいたします。
![287427511-54d5ba95-83b7-4546-9abd-245fcaa9e178](https://github.com/Yowkees/keyball/assets/143698349/1b029ab9-32b7-43f9-9316-599ce4379be0)

**Comment by Zazagann** [comment]
早速ご対応いただきありがとうございます！

該当箇所を繋いでみたら、おそらく左判定はできてそうなのですが、左のキーボードの一番左の列のキー（「Esc」「Tab」「Shift」）が依然として反応しないです。
どこの箇所の半田付けに問題があるのか教えていただきたいです。

よろしくお願いします。

**Comment by Ryo624** [comment]
ご対応していただきましてありがとうございます。

縦一列の反応がないということで、おそらくProMicro周辺のはんだ付けが原因の可能性があります。
現在反応がないキーはそれぞれCOL０という端子につながっていますので、ProMicroのCOL０と書かれた場所のはんだ付けをキーソケットのはんだ付けと併せて見直していただけますと幸いです。

ダイオードのはんだ付けが原因となる可能性もありますので、解決しない場合は再度ご連絡ください。

**Comment by Zazagann** [comment]
ご対応くださりありがとうございます。

ProMicroを右と左で入れ替えてみても、症状は全く変わらなかったので、COL 0 の12Pピンソケットのはんだをやり直してみましたが、それでもうまくいきませんでした。

ダイオードのはんだ付けや他に原因がある場合、どうすればよろしいでしょうか。
よろしくお願いします。

**Comment by Ryo624** [comment]
ProMicroの入れ替えなども実施していただきましてありがとうございます。

症状に変わりないということで了解しました。
添付しました写真の色どうしのキーソケット、ダイオードのはんだ付けの見直しをお願いいたします。

![LEFTパッド剥がれ](https://github.com/Yowkees/keyball/assets/143698349/26b066cf-e69a-45ac-b63b-8f51d27537e2)

**Comment by Ryo624** [comment]
追加でご確認いただきたいことがありご連絡させていただきます。

添付しました写真のキーソケットの赤点とProMicroの赤点（COL0の端子）の部分が導通しているかの確認をお願いできますでしょうか。

![LEFTパッド剥がれ](https://github.com/Yowkees/keyball/assets/143698349/bed73107-4b2d-4a01-8135-77e85abe0e91)

**Comment by Zazagann** [comment]
ご連絡が遅くなり申し訳ございません。
該当のキーソケット、ダイオードのはんだ付けを見直し、ダイオードに関しては部品を変えてみたのですが状況は改善しませんでした。

テスターを用いて導通の確認をしてみたところ、他のキーソケット部は対応するcolと導通が確認されたのですが、反応しない三か所のキーソケットの該当部では導通が確認されませんでした。

どのように対応すればよろしいでしょうか。よろしくお願いします。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

キーソケット、ダイオードのはんだ付けの見直し、導通確認を実施していただきましてありがとうございます。

一度、不具合が起きています実機を確認させていただきたいと思います。
キーボードの送付方法、送付先住所等についてやりとりを進めたいので、白銀ラボのウェブショップにあります問い合わせフォームからご連絡いただけますと幸いです。
